mlflow/pyfunc/loaders/chat_model.py

```diff
@@ -0,0 +1,106 @@
+import os
+from typing import Any, Dict, Optional
+
+import cloudpickle
+
+import mlflow
+from mlflow.exceptions import MlflowException
+from mlflow.pyfunc.model import (
+    CONFIG_KEY_ARTIFACT_RELATIVE_PATH,
+    CONFIG_KEY_ARTIFACTS,
+    CONFIG_KEY_CLOUDPICKLE_VERSION,
+    CONFIG_KEY_PYTHON_MODEL,
+    PythonModelContext,
+)
+from mlflow.types.llm import ChatMessage, ChatParams, ChatRequest, ChatResponse
+from mlflow.utils.annotations import experimental
+from mlflow.utils.model_utils import _get_flavor_configuration
+
+
+def _load_pyfunc(model_path: str, model_config: Optional[Dict[str, Any]] = None):
```

Comment:
This looks the same as PythonModel's _load_pyfunc function (except the wrapper it returned), could we reuse the function and extract the final class as a parameter?

Reply:
refactored the common part to `_load_context_model_and_signature`